### PR TITLE
Images Redesign

### PR DESCRIPTION
**Pull request by cmckee-dev**
## Images Redesign

**Index:**  

images-index

---

**New:**  

images-new

---
**Edit with small image preview:**  

image-edit-icon-size

**Edit with large image preview:**  
image-edit-wide-size

### CODE REVIEW

File: app/javascript/controllers/image_uploader_controller.js [hunk 1]
```diff
@@ -11,6 +12,7 @@ export default class extends Controller {
 
   connect () {
     this.progressBars = {}
+    console.log("Test");
```
Comment (andrewmcodes) [inline app/javascript/controllers/image_uploader_controller.js]: Can you remove this?

File: app/javascript/controllers/image_uploader_controller.js [hunk 2]
```diff
@@ -31,6 +33,7 @@ export default class extends Controller {
   }
 
   choose (event) {
+    console.log("Test 2");
```
Comment (andrewmcodes) [inline app/javascript/controllers/image_uploader_controller.js]: Can you remove this?

Comment (cmckee-dev) [reply app/javascript/controllers/image_uploader_controller.js]: :heavy_check_mark:

File: app/javascript/controllers/image_uploader_controller.js (hunk 1 above)
Comment (cmckee-dev) [reply app/javascript/controllers/image_uploader_controller.js]: :heavy_check_mark:

**Pull request merged**